### PR TITLE
Python script

### PR DESCRIPTION
**Pull request by schuhmaj**
## Notice

I will add some more options very soon. Nevertheless, I already open this Pull Request for some early feedback :)

## Possible Plots
All plots for the Iridium Cosmos 2 Configuration
First plot very similar to Orginal Paper Fig. 6 (page 6), but with 8 times as much fragments and some more bins
iridium_cosmos_am
iridium_cosmos_am_cum
iridium_cosmos_lc
iridium_cosmos_lc_cum
iridium_cosmos_scatter_lc_am

### CODE REVIEW

File: script/breakupScript.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import sys
+
+import numpy
+import numpy as np
+import scipy as sp
```
Comment (gomezzz) [inline script/breakupScript.py:5]: Don't think you are using this import?

File: script/breakupScript.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import pandas as pd
+
+
+def histogram_lc(data, title, cumulative):
```
Comment (gomezzz) [inline script/breakupScript.py:11]: For nice resolution I would recommend creating a figure first and setting the dpi on it with something like
`fig = plt.figure(figsize=(4,4),dpi=300)`

This create a 4 by 4 inch figure dpi 300. For print 300 or 600 is usually recommended, for presentations etc. 150.

File: script/breakupScript.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    weights = np.ones_like(lc) / float(len(lc))
+    plt.hist(lc, density=False, bins=1000, color='b', weights=weights, cumulative=cumulative)
+    plt.xscale("log")
+    plt.xlabel('$L_c [m]$')
+    plt.ylabel('Relative Probability')
+    if cumulative:
+        plt.title(title + ' $L_c$ Cumulative Distribution')
+    else:
+        plt.title(title + ' $L_c$ Distribution')
+    plt.show()
```
Comment (gomezzz) [inline script/breakupScript.py:22]: If you want you can also directly save the plots with `plt.savefig(filename,dpi=300)` if you want to

File: script/breakupScript.py [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    plt.xlabel('$\log(A/M) [m^2/kg]$')
+    plt.ylabel('Relative Probability')
+    if cumulative:
+        plt.title(title + ' $A/M$ Cumulative Distribution')
+    else:
+        plt.title(title + ' $A/M$ Distribution')
+    plt.show()
+
+
+def scatter_lc_am(data, title):
```
Comment (gomezzz) [inline script/breakupScript.py:40]: if you want to be super correct you can add docstrings see e.g. [here](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html).

File: script/breakupScript.py [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+    plt.xlabel('$L_c [m]$')
+    plt.ylabel('Relative Probability')
+    if cumulative:
+        plt.title(title + ' $L_c$ Cumulative Distribution')
+    else:
+        plt.title(title + ' $L_c$ Distribution')
+    plt.show()
+
+
+def histogram_am(data, title, cumulative):
```
Comment (gomezzz) [inline script/breakupScript.py:25]: just fyi there also typehints in python now, with e.g. 

`def histogram_am(data: pd.DataFrame,, title: str, cumulative: bool):`

(a bit overkill for this simple script, just fyi)

**Review by gomezzz — APPROVED** [review]
Only looked at .py , I think that is all for this PR, right?

File: script/breakupScript.py (hunk 1 above)
Comment (schuhmaj) [reply script/breakupScript.py:5]: Yes, that's right. It is removed.

_Before I followed the "Design guideline": "Viel hilft viel"_ :)

**Pull request merged**